README.md

```diff
@@ -0,0 +1,6 @@
+This is how I tried to do the project(https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/landing-page).
+I am open to your comments and suggestions.
+
```

Comment:
Please add a descriptive readme file that shows the project you're trying to do. A descriptive readme makes your project stand out.

Reply:
I love this review ❤️. It sounds polite, clear and easy to understand to the person you are reviewing 💪🏻.